src/components/Footer/MainWrap/MainWrap.scss

```
@@ -0,0 +1,24 @@
+@import '../../../styles/variable.scss';
+
+.footerWrap {
+  width: 90%;
+  font-size: 14px;
+}
+
+.footerLinks {
+  @include flex-space-between;
+}
+
+.copyright,
+.copy {
+  color: #535353;
+  font-weight: 300;
+}
+
+.copyright {
+  padding-top: 69px;
+}
+
+.copy {
+  padding-top: 60px;
+}
```

수연님 scss 파일 전체적으로 네스팅 적용 안되어있습니다!
컴포넌트의 최상위 태그이름을 컴포넌트 이름과 동일하게 지어주시고 해당 클래스네임으로 전체 scss를 네스팅해주셔야지 다른 컴포넌트들에 영향을 미치지 않습니다!
```suggestion
.MainWrap {
  .footerWrap {
    width: 90%;
    font-size: 14px;
  }

  .footerLinks {
    @include flex-space-between;
  }

  .copyright,
  .copy {
    color: #535353;
    font-weight: 300;
  }

  .copyright {
    padding-top: 69px;
  }

  .copy {
    padding-top: 60px;
  }
}
```